sdk/loadtestservice/loadtestservice/samples-dev/typescript/sample.ts

```diff
@@ -0,0 +1,63 @@
+import AzureLoadTesting, { AzureLoadTestingClient } from "@azure-rest/loadtestservice";
```

Comment:
@joheredi is this directory structure created by the generator? I thought all samples should be in the root of the `samples-dev` directory.

Reply:
I don't think codegen creates this structure

Reply:
I created this sample, not through codegen, so I have corrected the structure now.